packages/mux-player/src/index.ts

```diff
@@ -458,7 +493,7 @@ class MuxPlayerElement extends VideoApiElement implements MuxPlayerElement {
         return;
       }
 
-      const { dialog, devlog } = getErrorLogs(error, false);
+      const { devlog } = getErrorLogs(error, false);
```

Comment:
nitpick: the `dialog` and `devlog` code are already broken up into separate functions in the `errors.ts` module. Since their usage is no longer colocated, we should just export and use those directly (feel free to rename given their current verbosity).